app/scripts/minmax/dimMinMax.controller.js

```diff
@@ -100,23 +108,26 @@
 
     function getBuckets(items) {
       // load the best items
+      var types = ['Helmet','Gauntlets','Chest','Leg','ClassItem','Artifact','Ghost'];
       return {
         helmet: items.filter(function(item) { return item.type === 'Helmet'; }),
         gauntlets: items.filter(function(item) { return item.type === 'Gauntlets'; }),
         chest: items.filter(function(item) { return item.type === 'Chest'; }),
         leg: items.filter(function(item) { return item.type === 'Leg'; }),
         classItem: items.filter(function(item) { return item.type === 'ClassItem'; }),
         artifact: items.filter(function(item) { return item.type === 'Artifact'; }),
-        ghost: items.filter(function(item) { return item.type === 'Ghost'; })
+        ghost: items.filter(function(item) { return item.type === 'Ghost'; }),
+        all:  items.filter(function(item) { return types.indexOf(item.type) >= 0; })
       };
     }
 
     function initBuckets(items) {
       return {
         titan: getBuckets(items.filter(function(item) { return item.classType === 0 || item.classType === 3; })),
         hunter: getBuckets(items.filter(function(item) { return item.classType === 1 || item.classType === 3; })),
-        warlock: getBuckets(items.filter(function(item) { return item.classType === 2 || item.classType === 3; }))
-      };
+        warlock: getBuckets(items.filter(function(item) { return item.classType === 2 || item.classType === 3; })),
+        all: getBuckets(items.filter(function(item) { return [0,1,2,3].indexOf(item.classType) >= 0; }))
```

Comment:
don't need to filter if we want everything, maybe something like- `all: getBuckets(items)`?